platforms/android/library/src/main/java/io/element/android/wysiwyg/utils/StyleConfig.kt

```diff
@@ -1,10 +1,17 @@
 package io.element.android.wysiwyg.utils
 
+import android.graphics.drawable.Drawable
 import androidx.annotation.Px
 
 internal data class StyleConfig(
     @Px
     val bulletGapWidth: Float,
     @Px
     val bulletRadius: Float,
+    val inlineCodeHorizontalPadding: Int,
+    val inlineCodeVerticalPadding: Int,
```

Comment:
Maybe we could annotate these with `@Dimension(unit=Dimension.DP)` to differentiate them from the ones with `@Px` above?

Reply:
Actually these are also pixel values (just rounded to the nearest integer): [`b0388b0`](https://github.com/matrix-org/matrix-rich-text-editor/pull/454/commits/b0388b0544249de8dd8f1841340062392aeb3b1a)